pluggable_membrane.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	var eventingService sdk.EventService = nil
 	var gatewayService sdk.GatewayService = nil
 	var keyValueService sdk.KeyValueService = nil
-	var queueService sdk.QueueService = nil
+	var QueueService sdk.QueueService = nil
```

Comment:
Is there a reason for leading uppercase character ?

Reply:
find and replace error from a rename